R/get_rcode.R

```diff
@@ -189,11 +183,7 @@ get_datasets_code <- function(datanames, datasets) {
     "\n\n"
   )
 
-  str_filter <- teal.slice::get_filter_expr(datasets, datanames)
-  if (str_filter != "") {
-    str_filter <- paste0(str_filter, "\n\n")
```

Comment:
so we are losing this - does that matter?

Reply:
tbh probably not

Reply:
I tested with empty filters, no problem.